e2e/cypress/integration/interactive_menu/basic_options_spec.js

```diff
@@ -154,6 +154,24 @@ describe('Interactive Menu', () => {
         });
     });
 
+    it('IM21042 - "No items match" feedback', () => {
+        const missingUser = Date.now();
```

Comment:
I couldn't think of how else to get a username that is guaranteed to not exist. Is this OK?

Reply:
Yeah, that's a good choice.